source/robot_model/src/Model.cpp

```diff
@@ -39,6 +30,17 @@ Model::Model(const Model& model) :
 
 Model::~Model() {}
 
+Model& Model::operator=(const Model& model) {
+  this->robot_name_ = model.robot_name_;
+  this->urdf_path_ = model.urdf_path_;
+  this->alpha_ = model.alpha_;
+  this->epsilon_ = model.epsilon_;
+  // initialize the model and the solver
+  this->init_model();
+  this->init_qp_solver();
+  return (*this);
+}
+
```

Comment:
is there a reason for this change? looks like the function was just moved but not edited

Reply:
I just found it nicer to have the operator implementation above the first method implementation such that the methods are grouped